src/components/localization/withLocalisation.jsx

```diff
@@ -25,15 +25,9 @@ export const setLocaleMessages = (args) => {
 // If browser doesn't support Intl (i.e. Safari), then we manually import
 // the intl polyfill and locale data.
 if (!window.intl) {
-  require.ensure([
```

Comment:
I removed `require.ensure` because it is a webpack feature, and this module is not being bundled by webpack